pkg/mimirpb/timeseries_test.go

```diff
@@ -421,46 +421,53 @@ func TestPreallocTimeseries_SetLabels(t *testing.T) {
 		},
 		marshalledData: []byte{1, 2, 3},
 	}
-	p.SetLabels(FromLabelsToLabelAdapters(labels.FromStrings("__name__", "hello", "lbl", "world")))
+	expected := []LabelAdapter{{Name: "__name__", Value: "hello"}, {Name: "lbl", Value: "world"}}
+	p.SetLabels(expected)
 
-	require.Equal(t, []LabelAdapter{{Name: "__name__", Value: "hello"}, {Name: "lbl", Value: "world"}}, p.Labels)
+	require.Equal(t, expected, p.Labels)
 	require.Nil(t, p.marshalledData)
 }
 
 func BenchmarkPreallocTimeseries_SortLabelsIfNeeded(b *testing.B) {
-	bcs := []int{10, 100, 1_000, 10_000, 100_000, 1_000_000}
+	bcs := []int{10, 40, 100}
 
 	for _, lbCount := range bcs {
 		b.Run(fmt.Sprintf("num_labels=%d", lbCount), func(b *testing.B) {
-			// Generate unordered labels set.
+			// Generate labels set in reverse order for worst case.
```

Comment:
nit: It's not strictly reverse order (as final ordering will be: `lbl_1, lbl_10, lbl_11, ...`)